.github/BOTMETA.yml

```diff
@@ -670,6 +670,10 @@ files:
     maintainers: Gaetan2907
   $modules/keycloak_clientscope.py:
     maintainers: Gaetan2907
+  $modules/keycloak_clientsecret_info.py:
+    maintainers: fynncfchen johncant
+  $modules/keycloak_clientsecret_regenerate.py:
+    maintainers: fynncfchen johncant
   $modules/keycloak_group.py:
     maintainers: adamgoossens
```

Comment:
You will need an entry in this file for the new module_utils as well (around line 282).

Reply:
Done, thanks